src/librustc/diagnostics.rs

```diff
@@ -1941,6 +1941,53 @@ you want. Example:
 ```
 "##,
 
+E0492: r##"
+A constant borrow which contains interior mutability was attempted. Erroneous
+code example:
+
+```
+#![feature(const_fn)]
+
+use std::cell::UnsafeCell;
+
+const A: UnsafeCell<usize> = UnsafeCell::new(1);
+const B: &'static UnsafeCell<usize> = &A;
+// error: cannot borrow a constant which contains interior mutability, create
+//        a static instead
+
+// or:
+struct C { a: UnsafeCell<usize> }
+
+const D: C = C { a: UnsafeCell::new(1) };
+const E: &'static UnsafeCell<usize> = &D.a;
+
+// or:
+const F: &'static C = &D;
+```
+
+To solve this error, either you don't borrow `UnsafeCell` type with a constant
+or you don't use `UnsafeCell` type, or you wrap the `UnsafeCell` in another
+type:
```

Comment:
either use statics instead, or avoid using the cell in the constant.

Don't suggest the unsafe option.